src/loaders/ObjectLoader.js

```diff
@@ -610,6 +617,94 @@ THREE.ObjectLoader.prototype = {
 
 			}
 
+			/*
+			 * Now SkinnedMesh could have two bone sets in its children,
+			 * one is made from geometry.bones in SkinnedMesh constroctor
+			 * and another one is made from data.children in this method.
+			 * These two bone sets correspond to the same bones.
+			 * The former would represent a default stand pose while
+			 * the latter would represent a pose updated by skinning.
+			 * To let SkinnedMesh be updated pose, here this logic copies
+			 * the latter one to the former one then removes the latter one.
+			 * If in the latter one there're objects which don't correspond to
+			 * the ones in the former one, they'll be moved to the former one.
+			 */
```

Comment:
There are three markets that ThreeJS target in my opinion: those learning to code 3D (where ThreeJS is very dominant) and those creating 3D projects (there is a lot of competition here) and those using ThreeJS as the renderer for other tools (such as our usage.)  The exporters do not need to be fixed to target those learning to code 3D, they are happy with OBJ/MTL imports for the most part.  For most third party tools using ThreeJS as the renderer, they also do not need to be fixed, these other tools have their own loaders -- these users of ThreeJS Iview as low value to the project as they tend not to be contributors back to the project though because they view the other users of ThreeJS as potential competitors.  But those creating serious 3D projects need to be able to rely on their artists (who want to work in their content creation tool of choice since that is where they are productive) creating content that just works in ThreeJS.  If there are non-trivial post export steps that need to be done to get content to work, when it will just work in Unity3D or other tools, then you are putting ThreeJS at a disadvantage, and a serious one at that.  A great model to follow is Blend4Web - it is nearly perfect WYSIWYG when moving from Blender to WebGL both in terms of materials, lights, animations, meshes, bones, skinning, etc.